prow/knative/cluster/monitoring/secrets/alertmanager-prow_secret.yaml

```diff
@@ -32,11 +32,16 @@ stringData:
     receivers:
     - name: 'email-alerts'
       email_configs:
-      - to: 'chaodai+prow-alert@google.com'
+      - to: 'k8s-infra-oncall+prow-alert@google.com'
```

Comment:
k8s-infra-oncall is not appropriate for these instances. We should use our oncall group or osp-engprod-tools instead.

Reply:
Done